package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -604,20 +605,18 @@ public void ExportOverrides(Scene sceneInWhichToStoreTransforms)
                     overs,
                     BasisTransformation.SlowAndSafe,
                     exportUnvarying: false,
-                    zeroRootTransform: true);
-
-                var rel = ImporterBase.MakeRelativePath(overs.FilePath, sceneToReference.usdFullPath);
-                GetFirstPrim(overs).GetReferences().AddReference(rel, GetFirstPrim(baseLayer).GetPath());
```

Comment:
@jcowles Do you remember why were adding a relative ref to the original prim? 
My view here is that until we provide tools to target changes to specific layers  I consider the override layer exported from Unity should not reference any other files. Any layering that needs to happen to use the overrides is out of scope and should be done by the user.

Note that I'm also removing the original layer l619